content/en/_index.html

```diff
@@ -158,7 +160,12 @@ <h4 class="section-text-bold">
                             </h4>
                         </div>
                         <p class="section-text">
-
+                            Python array library targeting conformance to core <a href="https://data-apis.org/array-api/latest/">Python Array API</a> 
+                            specification.   
+                            </p>
+                            <p>
+                            Implemented using DPC++ <a href="https://intelpython.github.io/dpctl/latest/docfiles/dpctl/dpctl.tensor_pyapi.html#dpctl-tensor-pyapi">dpctl.tensor</a> lets Python users get their job done using tensor operations powered by 
```

Comment:
<a href="https://intelpython.github.io/dpctl/latest/docfiles/dpctl/dpctl.tensor_pyapi.html#dpctl-tensor-pyapi">dpctl.tensor</a> is a Python native extension library implemented using SYCL. The library lets Python users get their job done using tensor operations powered by